src/rep/mesh_deprecated.jl

```diff
@@ -0,0 +1,238 @@
+export TriMesh,
```

Comment:
Maybe don't keep this file altogether. If we need to see the way it was originally done we always have the git history